### PR TITLE
Boyer Moore Algorithm in C

### PR DESCRIPTION
**Pull request by barnamensch056**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by barnamensch056** [comment]
Please review my code.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Remove ```.vscode/settings.json``` and  ```Python/graphs/graph.png```. Update the README.md file.

**Comment by barnamensch056** [comment]
Is this ok now, Sir?

File: C/cp/BoyerMooreVote.c [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#include<stdio.h>
```
Comment (HarshCasper) [inline C/cp/BoyerMooreVote.c]: Add a description/pitch about the Algorithm that you have implemented

File: C/cp/BoyerMooreVote.c [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        else
+        {
+         if(temp==ar[i])
+         {
+             count+=1;
+         }
+         else
+         {
+             count-=1;
+         }
```
Comment (HarshCasper) [inline C/cp/BoyerMooreVote.c]: Format the Code in and around the Operators

File: C/cp/BoyerMooreVote.c [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        count2+=1;
+  }
+  if(count2>len/2)
+     bmv.majority_check=1;
+  else
+     bmv.majority_check=0;
+return bmv;
+}
+
+//diver function
```
Comment (HarshCasper) [inline C/cp/BoyerMooreVote.c]: Minor Typo. It should be `driver` not `diver`

File: C/cp/BoyerMooreVote.c [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+         {
+             count-=1;
+         }
+            
+            
+        }
+        /*algorithm used to find the majoritty number by taking a temporary
+        variable which changes when count of it decreases to 0 with other numbers in the
+        list and increases when the corresponding number matches,resulting to the last number with +ve 
+        value will become the majority number*/
```
Comment (HarshCasper) [inline C/cp/BoyerMooreVote.c]: Make the Comments more consistent and grammatically correct

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by barnamensch056** [comment]
Sure I'll make the changes

File: C/cp/BoyerMooreVote.c (hunk 1 above)
Comment (HarshCasper) [reply C/cp/BoyerMooreVote.c]: Not done yet. Kindly take a look at other Code Files to better understand the requirement.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please format the Code as well.

**Comment by barnamensch056** [comment]
Sure I'll look into the matter

**Comment by HarshCasper** [comment]
Are you still working on it @barnamensch056 ?

**Comment by barnamensch056** [comment]
Yeah I am learning the clang format

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**